etc/entrypoint.sh

```diff
@@ -27,6 +27,33 @@ if ! whoami &> /dev/null; then
   fi
 fi
 
+providePreliminaryConfiguration() {
```

Comment:
maybe use plain files instead of cat/echo ? will be easier to maintain
and we could provide these files in `/JetBrains/default` (or another path) from Dockerfile

Reply:
Yeah, we can, but not in /JetBrains directory, as this directory overwrites with volume mount. I'll try to do that with different path

Reply:
sure, can be `/etc/default/jetbrains` or anywhere else

Reply:
Got it. I'll provide an update in next commit